examples/src/boids/boid.rs

```diff
@@ -60,7 +60,7 @@ pub fn game_tick(dt: f32, cursor: mint::Point2<f32>, boid: &mut Boid, other_boid
 
     // unwrap bt for boid
     let mut bt = boid.bt.clone();
-    let db = &*bt.get_blackboard().get_db();
+    let db = &*bt.get_blackboard();
     let win_width: f32 = *db.get("win_width").unwrap();
     let win_height: f32 = *db.get("win_height").unwrap();
```

Comment:
Nit: Can you check here if the &* which is first a reference (*) followed by a dereference (&) on line 63, followed by a reference again (*) on line 64 and 65 is redundant? I think this can be simplified

Reply:
Done. Also found another case in the 3d example.